nbs/models.moment.ipynb

```diff
@@ -0,0 +1,1518 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #111.            model = MOMENTPipeline.from_pretrained(
</pre><p>perhaps instead of using <code>model</code> we can use <code>moment</code> to prevent confusion with the <code>model</code> parameter defined above. maybe, it might be a good idea to rename that parameter to <code>mode_name</code>.     </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1011/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Ahh yes, you are very right